### DISCUSSION TITLE
Is it possible to grab a user's name out of the Auth collection?

### DISCUSSION THREAD

**Discussion by johnnynumberfive-coder** [pocketbase/pocketbase#3492] [Q&A]
I have an app with two collections (users, and projects).  The projects collection has a relation field called user, and a relation field called username.  Using project.user returns the id, but so does project.username.  How would I return either the user's username, or better yet the user's name?

**Comment by tigawanna** [comment]
https://pocketbase.io/docs/working-with-relations/

You'll have to expand the user field and get it under , response.expand.user
